app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,33 @@
+class RegistrationsController < Devise::RegistrationsController
+
+  def create
+    build_resource(sign_up_params.merge(user_type: params[:user_type]))
+    resource.save
+    yield resource if block_given?
+    if resource.persisted?
+      if resource.active_for_authentication?
+        set_flash_message! :notice, :signed_up
+        sign_up(resource_name, resource)
+        respond_with resource, location: after_sign_up_path_for(resource)
+      else
+        set_flash_message! :notice, :"signed_up_but_#{resource.inactive_message}"
+        expire_data_after_sign_in!
+        respond_with resource, location: after_inactive_sign_up_path_for(resource)
+      end
+    else
+      clean_up_passwords resource
+      set_minimum_password_length
+      respond_with resource
+    end
+  end
+
+  protected
+
+  def after_sign_up_path_for(resource)
+    new_orphanage_path
+  end
+
+  def sign_up_params
+    params.require(:user).permit(:type, :email, :password, :password_confirmation)
```

Comment:
I missed to update this and since everything is working fine I'm inclined to think we don't need this method. Delete?